config/recipe.yml

```diff
@@ -32,13 +32,18 @@ modules:
 
   - type: bling # configure what to pull in from ublue-os/bling
     install:
-      - justfiles # add "!include /usr/share/ublue-os/just/bling.just"
+      - justfiles # add "!include /usr/share/ublue-os/just/100-bling.just"
                   # in your custom.just (added by default) or local justfile
-      - nix-installer # shell shortcuts for determinate system's nix installers
+      - nix-installer # these are the silverblue nix installer scripts from dnkmmr69420
       - ublue-os-wallpapers
       # - ublue-update # https://github.com/ublue-os/ublue-update
+      # - 1password # 1Password (stable) and `op` CLI tool installed on base image
```

Comment:
Reading this comment again, I think it'd be more grammatical and correct as ``# install 1Password (stable) and `op` CLI tool`` (which, again, is a fault in the bling repository and not this one)